src/engine/runtime.js

```diff
@@ -1532,6 +1542,21 @@ class Runtime extends EventEmitter {
         };
     }
 
+    /**
+     * Helper for _convertPlaceholdes which handles variable fields which are a specialized case of block "arguments".
+     * @param {object} argInfo Metadata about the inline image as specified by the extension
+     * @return {object} JSON blob for a scratch-blocks variable field.
+     * @private
+     */
+    _constructVariableJson (argInfo, placeholder) {
+        return {
+            type: 'field_variable',
+            name: placeholder,
+            variableTypes: [argInfo.variableType] ?? [''],
```

Comment:
[...] is always non-nullish so ?? is doing nothing here

Reply:
Fixed, the new system _intends_ to allow an input of either "list", ["list"], or ["list", "broadcast_msg"], so you can list all of them at once (if you want to for some reason